data-prepper-api/src/main/java/com/amazon/dataprepper/model/event/Event.java

```diff
@@ -85,4 +86,10 @@
      * @since 1.2
      */
     boolean isValueAList(String key);
+
+    /**
+     * @return a Map representation of the Event
+     * @since 1.3
+     */
+    Map<Object, Object> getAsMap();
```

Comment:
Our model guarantees that the key is a string. So, I think we can return `Map<String, Object>` here instead.